### PR TITLE
Error handling sanitize bug

### PR DESCRIPTION
**Pull request by frederik-sandfort1**
Fixes an error that MolSanitizeErrors are caught but not treated correctly by ErrorHandling

- fixes one line of code (replace wrong return)
- adds a dummy test for checking
- automatic isort on some other files

### CODE REVIEW

File: test_extras/test_chemprop/test_models.py [hunk 1]
```diff
@@ -28,10 +28,10 @@
 # pylint: disable=relative-beyond-top-level
 from test_extras.test_chemprop.chemprop_test_utils.compare_models import compare_params
 from test_extras.test_chemprop.chemprop_test_utils.constant_vars import (
-    NO_IDENTITY_CHECK,
```
Comment (c-w-feldmann) [inline test_extras/test_chemprop/test_models.py:31]: @JochenSiegWork I think we need to check our isort settings

**Review by c-w-feldmann — APPROVED** [review]
LGTM

File: molpipeline/pipeline/_molpipeline.py [hunk 2]
```diff
@@ -349,7 +349,7 @@ def transform_single(self, input_value: Any) -> Any:
                 elif isinstance(p_element, FilterReinserter):
                     iter_value = p_element.transform_single(iter_value)
             except MolSanitizeException as err:
-                return InvalidInstance(
+                iter_value = InvalidInstance(
```
[the remark this answers is not in the document]
Comment (frederik-sandfort1) [reply molpipeline/pipeline/_molpipeline.py:352]: Jap... But not sure where this came from and why it was there at all...

File: test_extras/test_chemprop/test_models.py (hunk 1 above)
Comment (frederik-sandfort1) [reply test_extras/test_chemprop/test_models.py:31]: but it went through? I just ran 'isort . --profile black' with latest isort...

Comment (c-w-feldmann) [reply test_extras/test_chemprop/test_models.py:31]: Yes, the issue is, that apparently in the last PR the imports were not in the correct order, but the pipeline still passed.

Comment (c-w-feldmann) [reply test_extras/test_chemprop/test_models.py:31]: Created [issue](https://github.com/basf/MolPipeline/issues/87)

**Pull request merged**